introduction_to_amazon_algorithms/jumpstart-foundation-models/jumpstart-text-generation-inference.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>This is my own preference: since we want to make the notebook simple and short. Do you think we can remove these dash dash line on each MD cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
changed